app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx

```diff
@@ -94,17 +100,29 @@ function ProtocolSetupStep({
             name={status === 'ready' ? 'ot-check' : 'ot-alert'}
           />
         ) : null}
-        <StyledText as="h4" fontWeight={TYPOGRAPHY.fontWeightSemiBold}>
+        <StyledText
+          as="h4"
+          fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+          color={isActionUnavailable ? COLORS.darkBlack60 : COLORS.darkBlack100}
+        >
           {title}
         </StyledText>
         <Flex flex="1" justifyContent={JUSTIFY_END}>
-          <StyledText as="p" textAlign={TEXT_ALIGN_RIGHT}>
+          <StyledText
+            as="p"
+            textAlign={TEXT_ALIGN_RIGHT}
```

Comment:
I think if you don't want to wait for CIs, you can include this in a following PR.

```suggestion
            textAlign={TYPOGRAPHY.textAlignRight}
```

Reply:
oh good catch, ive been trying to remember to use the right constant when i add code but need to be better about changing existing instances of it. Maybe it can be done in a follow up ticket